ThingIFSDK/ThingIFSDK/ScheduleOncePredicate.swift

```diff
@@ -25,12 +25,11 @@ public class ScheduleOncePredicate: NSObject,Predicate {
     }
 
     public required init(coder aDecoder: NSCoder) {
```

Comment:
NSDate is conforms to NSSecureCoding.
Why not just use NSDate capability for encoding/decoding?